external/source/exploits/CVE-2022-34918/src/util.c

```diff
@@ -0,0 +1,188 @@
+#define _GNU_SOURCE
+#include <stdint.h>
+#include <stdlib.h>
+#include <sched.h>
+#include <stdio.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <string.h>
+#include <sys/utsname.h>
+
+#include "log.h"
+#include "util.h"
+
+struct kernel_info kernels[] = {
+// 22.04 LTS
+    { "5.15.0-24-lowlatency #24-Ubuntu", 0x3e68a0, 0x3e7690, 0x1e8c320 },
```

Comment:
Maybe add a comment about what functions those offsets are referring to, to make it easy to add more.

Reply:
I added a comment. The structure is declared in: https://github.com/rapid7/metasploit-framework/blob/011f0ac9900e415dda4422f35a33f67ef9579ab1/external/source/exploits/CVE-2022-34918/inc/util.h#L10-L15